src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Ball ball = new Ball();
+
+    public Ball getRandomBall() {
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
+        return ball;
+    }

```

Comment:
100 should be a class constant, don't forget an appropriate access modifier
Random and Color Supplier should be class fields
Ball OTOH should be only a local variable